tests/vehicle_list.yaml

```
@@ -0,0 +1,25 @@
+- inputText: "ZG1234BB"
+- pressKey: "Enter"
+- waitForAnimationToEnd
+
+# verify search results
+- assertVisible: "ZG1234BB"
+- assertVisible: "Golf"
+- assertVisible: "Car"
+- assertVisible: "6-07-18"
+- assertVisible: "B"
+- assertVisible: "Test3 Osoba3"
+- assertVisible: "6-07-19" 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)